projects/tinkerbell/pbnj/docker/linux/Dockerfile

```diff
@@ -0,0 +1,56 @@
+ARG BASE_IMAGE # https://gallery.ecr.aws/eks-distro-build-tooling/eks-distro-minimal-base
+ARG BUILDER_IMAGE
+
+FROM $BUILDER_IMAGE as ipmitool-builder
+
+ARG IPMITOOL_CLONE_URL
+ARG IPMITOOL_GIT_TAG
+
+WORKDIR /newroot
+
+RUN yum install -y \
+    autoconf \
+    automake \
+    git \
+    libtool \
+    make \
+    readline-devel
+
+RUN git clone $IPMITOOL_CLONE_URL && \
+    cd ipmitool && \
+    git checkout $IPMITOOL_COMMIT && \
+    ./bootstrap && \
+    ./configure \
+        --prefix=/usr/local \
+        --enable-ipmievd \
+        --enable-ipmishell \
+        --enable-intf-lan \
+        --enable-intf-lanplus \
+        --enable-intf-open && \
+    make && \
+    make install
+
+RUN set -x && \
```

Comment:
Can you manually run `ldd /usr/local/bin/ipmitool` and post the output here?  Just want to make sure we got the runtime deps right

Reply:
Does something you install below bring in zlib as well?

Reply:
The zlib library seems to be already installed on that image.